versions.tf

```
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 3.69"
+      version = ">= 4.2.0"
```

```suggestion
      version = ">= 3.75.0"
```
Hi @antonbabenko 
Thanks for your work on this.
Actually, Hashicorp "backported" those breaking changes for S3 resources into AWS provider version `3.75.0`.
So I think this is a clear way to keep the code with `3.x` release unless all major modules will support `4.x` in full.

---

Yes, I will incorporate this change also. Thank you!